src/python/pants/backend/jvm/tasks/jvm_platform_analysis.py

```diff
@@ -30,7 +31,10 @@ def _is_jvm_target(cls, target):
 
   @classmethod
   def jvm_version(cls, target):
-    return target.platform.target_level
+    if target.platform:
+      return target.platform.target_level
+    else:
```

Comment:
Unnecessary else due to early `return`

Reply:
I feel like I've tried very hard to write this kind of code especially coming from a lisp background and have been explicitly advised that an if/else like this is somehow more clear. I have taken your comment as an excuse to do the right thing.

Reply:
I often prefer the else, but after being peppered enough with errors by Pylint I’ve trained myself to alway early return. 

I thought for some reason we were using Pylint, but it looks like we aren’t and we don’t have a consistent style guide established (John and I have discussed creating one) so this is okay to keep if you prefer.